### ISSUE TITLE
Accessibility Technical Analysis - Missing focus management when providing details about conditions

### ISSUE THREAD

**Issue by etanb** [department-of-veterans-affairs/va.gov-team#124238]
## What happened?

**Describe what went wrong**:

On the Supporting evidence screen, file upload.

Steps:

Upload a file that the system deems "forbidden" (unknown limitations, as this triggered only sometimes when uploading the same file) OR select a file that is not allowed (such as an .mp4)
The eror message is announced
Do nothing with the file and TAB to and activate "Continue"
Focus is moved to the file section in error, and there is a visual error treatment, but there is no indication programmatically or announcement to the screen reader, that something is in error. The error text (i.e. "Forbidden" is not read when the section receives focus.

verbatim from: https://github.com/department-of-veterans-affairs/va.gov-team/issues/117628

**Add a screen recording or photo**:

<img width="551" height="261" alt="Image" src="https://github.com/user-attachments/assets/94868291-5c23-49e9-b596-f8e922db94e7" />

## What is the desired behavior?
Ensure that the error state is indicated and announced to screen reader users.
